opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/aggregation.py

```diff
@@ -266,6 +295,7 @@ def collect(
         with self._lock:
             value = self._value
             self._value = None
+            exemplars = self._collect_exemplars()
```

Comment:
Perhaps we put collection of exemplars at the end of collection of the value so that the collection times do not account for processing of exemplars.